src/test/java/com/neuvector/ScannerTest.java

```diff
@@ -45,14 +57,30 @@ public void scanRegistryTest(){
         String repositoryTag = "3.6";
         String nvScannerImage = "neuvector/scanner:latest";
         String nvRegistryURL = registryURL;
-        String nvRegistryUser = "xxx";
-        String nvRegistryPassword = "xxx";
+        String nvRegistryUser = "";
+        String nvRegistryPassword = "";
         String mountPath = "/temp";  //mountPath is an optional parameter. It will use "/var/neuvector" by default.
 
         Registry registry = new Registry(registryURL, regUser, regPassword,repository,repositoryTag);
         NVScanner scanner = new NVScanner(nvScannerImage, nvRegistryURL, nvRegistryUser, nvRegistryPassword, mountPath);
 
         ScanRepoReportData scanReportData = Scanner.scanRegistry(registry, scanner, license);
+
+        File file = new File(mountPath + "/scan_result.json");
+        UserPrincipal user = getUserPrincipal(mountPath, file);
+
+        assertTrue(user.getName().equals("root") || user.getName().contains(System.getProperty("user.name")));
```

Comment:
What if the file exist and is owned by root? that's why I added this assertion here.

Reply:
This is a test so we do not need to ask _what if the file exists.._ File will not exist for tests.

Reply:
649bee926c91a7f35b45caff6b84f63640bbce86